crates/nu-cli/src/types/deduction.rs

```diff
@@ -765,60 +765,59 @@ impl VarSyntaxShapeDeductor {
                         )],
                     )?;
                 }
-                Operator::In | Operator::NotIn => {
-                    match var_side {
-                        BinarySide::Left => match &expr.expr {
-                            Expression::List(list) => {
-                                if !list.is_empty() {
-                                    let shapes_in_list = self
-                                        .get_shapes_in_list_or_insert_dependency(
+                Operator::In | Operator::NotIn => match var_side {
```

Comment:
Why is the deduction code changing? If you need to update it, may make sense to put it into a separate PR and just have this one focused on the date changes.

Reply:
This is changed by `cargo fmt --all`. When I submitted this PR, the Azure Pipelines checkes reported code formatting issues issues so I run that command to fix it. I didn't do any manual change on it. Should I submit a separate PR for this?

Reply:
That's interesting. Are you using the nightly compiler or the latest stable? Generally we use stable, but I've noticed that sometimes they do release formatting changes.

Reply:
I'm running `cargo 1.48.0 (65cbdd2dc 2020-10-14)`. I did some tests earlier: 

- check out the commit `e299e76fcf7a17dabc640a0ff6c02f53fa6548c6` ([commit link](https://github.com/nushell/nushell/commit/e299e76fcf7a17dabc640a0ff6c02f53fa6548c6)) , which is the last commit before my fork of the project
- run `cargo fmt --all -- --check`
- got the formatting issue report

**The current repo no longer have this problem because it has been fixed in a [recent commit](https://github.com/nushell/nushell/commit/b193303aa34fb648dd34068131de28fa7cc98fd9#diff-b6131e0fd615bf0e97bd318197f154553c07306adbe2dac3ef52750448eab430).**